.github/workflows/ci.yml

```diff
@@ -9,37 +9,42 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - name: Fetch the src
-      uses: actions/checkout@v3
-    - name: Set up Python 3.11
-      uses: actions/setup-python@v4
-      with:
-        python-version: 3.11
+      - name: Fetch the src
+        uses: actions/checkout@v3
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.11
 
-    - name: Install requirements
-      run: python -m pip install -r requirements/dev.in -c requirements/dev.txt
+      - name: Install requirements
+        run: python -m pip install -r requirements/dev.in -c requirements/dev.txt
 
-    - name: Run tests
-      run: python -m manage test
+      - name: Run tests
+        run: python -m coverage run manage.py test -v 2
```

Comment:
I like how https://adamj.eu/tech/2019/04/30/getting-a-django-application-to-100-percent-coverage/ integrates the coverage initialization into `manage.py`. This allows running the same simple command in CI and locally without having to duplicate the coverage wrapper invocation.

Although, I'd probably structure that as a context manager, making use of `@contextlib.contextmanager` and `contextlib.nullcontext()`. It'd add more elegance to the integration.

Reply:
Per https://coverage.readthedocs.io/en/latest/api_coverage.html, calling `Coverage.start()` and `Coverage.stop()` separately can be replaced by the `Coverage.collect()` context manager. So I'd use that too.